UITests/Tests/AxeScanAllTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.IO;
+using System.Collections.ObjectModel;
+
+namespace UITests.Tests
+{
+    [TestClass]
+    public class AxeScanAll : TestBase
+    {       
+        public static readonly string jsonUri = "ms-appx:///../../../../../../WinUIGallery/DataModel/ControlInfoData.json";
+        public static new WindowsDriver<WindowsElement> Session => SessionManager.Session;
+
+        public class ControlInfoData
+        {
+            public ObservableCollection<ControlInfoDataGroup> Groups { get; set; }
+        }
+
+        public class ControlInfoDataGroup
+        {
+            public string UniqueId { get; set; }
+            public string Title { get; set; }
+            public string Subtitle { get; set; }
+            public string Description { get; set; }
+            public string ImagePath { get; set; }
+            public string ImageIconPath { get; set; }
+            public string ApiNamespace { get; set; }
+            public bool IsSpecialSection { get; set; }
+            public string Folder { get; set; }
+            public ObservableCollection<ControlInfoDataItem> Items { get; set; }
+        }
+
+        public class ControlInfoDataItem
+        {
+            public string UniqueId { get; set; }
+            public string Title { get; set; }
+            public string ApiNamespace { get; set; }
+            public string Subtitle { get; set; }
+            public string Description { get; set; }
+            public string ImagePath { get; set; }
+            public string ImageIconPath { get; set; }
+            public string BadgeString { get; set; }
+            public string Content { get; set; }
+            public bool IsNew { get; set; }
+            public bool IsUpdated { get; set; }
+            public bool IsPreview { get; set; }
+            public bool HideSourceCodeAndRelatedControls { get; set; }
+            public ObservableCollection<ControlInfoDocLink> Docs { get; set; }
+            public ObservableCollection<string> RelatedControls { get; set; }
+
+            public bool IncludedInBuild { get; set; }
+        }
+
+        public class ControlInfoDocLink
+        {
+            public ControlInfoDocLink(string title, string uri)
+            {
+                this.Title = title;
+                this.Uri = uri;
+            }
+            public string Title { get; set; }
+            public string Uri { get; set; }
+        }
+
+        private static ControlInfoData controlInfoData;
+
+        private string[] ExclusionList =
+        {
+            "WebView2" // 46668961: Web contents from WebView2 are throwing null BoundingRectangle errors.
```

Comment:
I'm curious if this is an issue with WebView2, the page we show or Axe. In the last case, maybe we should open an issue on their repo?

Reply:
The issue is already open on their repo :) I should follow up with them actually.